internal/utils/httputil.go

```diff
@@ -65,8 +65,11 @@ func WriteJSONError(w http.ResponseWriter, errorCode, errorDescription string, s
 	w.Header().Set("Content-Type", "application/json")
 
 	w.WriteHeader(statusCode)
-	json.NewEncoder(w).Encode(map[string]string{
+	err := json.NewEncoder(w).Encode(map[string]string{
 		"error":             errorCode,
 		"error_description": errorDescription,
 	})
+	if err != nil {
+		return
```

Comment:
Shall we add a error log?

Reply:
Fixed with: https://github.com/asgardeo/thunder/compare/18e958c4fef32e306e250c33f53bc073aa9f2326..eb3fac7ab874ba7ae9b0aa2f62d0d3c931b27537